go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/spf13/viper v1.12.0
 	github.com/tendermint/tendermint v0.34.20
 	github.com/tendermint/tm-db v0.6.6
+	go.uber.org/zap v1.21.0
```

Comment:
This seems not to be the latest one. Why not use the v1.24.0?
https://github.com/uber-go/zap/releases

Reply:
Use version 1.24.0
https://github.com/datachainlab/yui-relayer/pull/4/commits/30d32e2c9d818ea9c7c8010c6aa047e7ccbe3cb5